packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -867,6 +895,9 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
 
   @override
   void updateEditingValue(TextEditingValue value) {
+    if(widget.readOnly) {
```

Comment:
Here is the best place I can think of to disable text update, I cannot cut off the listener from the keyboard since i have to deal with keyboard select

Reply:
This is probably worth a comment in the file, e.g. 

```
// Since we still have to support keyboard select, this is the best place to disable text updating.
```

Reply:
Can you add a test that ensures that selecting via keyboard continues to work?

Reply:
added 'Read only keyboard selection test' and added comment